pkg/database/tigris.go

```diff
@@ -0,0 +1,62 @@
+package database
+
+import (
+	"context"
+
+	"github.com/tigrisdata/tigris-client-go/tigris"
+
+	"github.com/rs/zerolog/log"
+
+	_ "github.com/golang-migrate/migrate/v4/database/mysql"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/golang-migrate/migrate/v4/source/github"
+	"github.com/warrant-dev/warrant/pkg/config"
+)
+
+type Tigris struct {
+	T      *tigris.Client
+	Config *config.TigrisConfig
+}
+
+func NewTigris(config *config.TigrisConfig) *Tigris {
+	return &Tigris{
+		Config: config,
+	}
+}
+
+func (ds *Tigris) Type() string {
+	return TypeTigris
+}
+
+func (ds *Tigris) Connect(ctx context.Context) error {
+	t, err := tigris.NewClient(ctx,
+		&tigris.Config{
+			URL:          ds.Config.URL,
+			ClientID:     ds.Config.ClientID,
+			ClientSecret: ds.Config.ClientSecret,
+			Project:      ds.Config.Project,
+		})
+	if err != nil {
+		return err
+	}
+
+	ds.T = t
+
+	log.Debug().Msgf("Connected to Tigris database %s", ds.Config.Project)
+
+	return nil
+}
+
+func (ds *Tigris) Migrate(_ context.Context, version uint) error {
+	//	tigris.SchemaVersion = version
```

Comment:
Is there anything remaining to be done here or can we remove this comment?

Reply:
Removing.